Newsletters: add 285 (2024-01-17)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Bitcoin Core 29058 @adamjonas 
- [x] Other merges @harding

Includes Ark topic that was previously in #1452

bitschmidty commented on _posts/en/newsletters/2024-01-17-newsletter.md:
```diff
@@ -0,0 +1,246 @@
+    The new vulnerability was discovered by Morehouse's following up on
+    his prior work on fake funding, which he also responsibly disclosed
+    (see [Newsletter #266][news266 lnbugs]).  When re-testing nodes that
+    had implemented fixes for fake funding, was able to trigger a [race
```
```suggestion
    had implemented fixes for fake funding, he was able to trigger a [race
```

bitschmidty commented on _posts/en/newsletters/2024-01-17-newsletter.md:
Maybe? 

```suggestion
    [joinpools][topic joinpools], reduced-signature [DLCs][topic dlc],
```

bitschmidty commented on _posts/en/newsletters/2024-01-17-newsletter.md:
We have a good explanation of taproot internal keys here: https://bitcoinops.org/en/newsletters/2019/05/14/#complex-spending-with-taproot

Perhaps that or another resource would be helpful for reader

bitschmidty commented on _posts/en/newsletters/2024-01-17-newsletter.md:
2019? https://bitcoinops.org/en/newsletters/2019/07/24/#bitcoin-core-15681

bitschmidty commented on _posts/en/newsletters/2024-01-17-newsletter.md:
Maybe reword to take out "we’d"

vostrnad commented on _posts/en/newsletters/2024-01-17-newsletter.md:
```suggestion
      also allow the users of contract protocols like LN to opt in to v3
```

vostrnad commented on _posts/en/newsletters/2024-01-17-newsletter.md:
```suggestion
  that contain more than 0 satoshis.  An ephemeral anchor pays to an output
  script equivalent to `OP_TRUE`, allowing anyone to spend it.
```
Personally I'd slightly rewrite this:
```suggestion
  that contain more than 0 satoshis.  An ephemeral anchor pays to a
  standardized output script that anyone can spend.
```

murchandamus commented on _posts/en/newsletters/2024-01-17-newsletter.md:
Ugh, congestion control is such an absurd application, I really don’t understand why people keep bringing that up.

harding replied:
It's still the first thing described in BIP119:

> This BIP proposes a new opcode, OP_CHECKTEMPLATEVERIFY, to be activated as a change to the semantics of OP_NOP4.
>
> The new opcode has applications for transaction congestion control and [...]

(But I also tend to think that it's impractical.)

murchandamus reviewed (commented):
Very cool newsletter so far.

vostrnad commented on _posts/en/newsletters/2024-01-17-newsletter.md:
```suggestion
- [LDK 0.0.119][] in a new release of this library for building
```

harding replied:
Thank you @vostrnad!  I just realized I missed the correction from LND to LDK.

azuchi commented on _posts/en/newsletters/2024-01-17-newsletter.md:
```suggestion
- [LDK 0.0.119][] is a new release of this library for building
```

harding replied:
Oof, thank you @azuchi !

azuchi commented on _posts/en/newsletters/2024-01-17-newsletter.md:
```suggestion
[ldk 0.0.119]: https://github.com/lightningdevkit/rust-lightning/releases/tag/v0.0.119
```

harding commented:
Made all edits (thanks @bitschmidty @vostrnad @murchandamus @azuchi !), added lede and releases, reviewed @adamjonas contribution (thanks!), and added topic entries.  Thanks everyone!
